python/paddle/audio/datasets/esc50.py

```diff
@@ -166,14 +168,14 @@ def __init__(
             files=files, labels=labels, feat_type=feat_type, **kwargs
         )
 
-    def _get_meta_info(self) -> List[collections.namedtuple]:
+    def _get_meta_info(self) -> list[collections.namedtuple]:
```

Comment:
```suggestion
    def _get_meta_info(self) -> list[meta_info]:
```

这里 `meta_info` 是个 `namedtuple` ，源代码

``` python
    meta_info = collections.namedtuple(
        'META_INFO',
        ('filename', 'fold', 'target', 'category', 'esc10', 'src_file', 'take'),
    )
```

其实有几个问题

- 应该是 `meta_info = collections.namedtuple('meta_info', ...` ，或者用 `class-base`
- 可以用 `typing.NamedTuple` 进行标注，然后，成员变量都是 `str` 类型

可以参考 

``` python
from __future__ import annotations

from typing import NamedTuple
import collections

class A(NamedTuple):
    a: int

b = collections.namedtuple('b', 'a')

class c(NamedTuple):
    a: int

def test(a: A, b: b, c: c) -> None:
    return



test(A(123), b(123), c(123))


class D:
    class e(NamedTuple):
        a: int

    def test(self)->list[e]:
        return [self.e(123)]
        
```